test/e2e/serial/tests/configuration.go

```diff
@@ -105,7 +105,7 @@ var _ = Describe("[serial][disruptive][slow] numaresources configuration managem
 	Context("cluster has at least one suitable node", func() {
 		timeout := 5 * time.Minute
 
-		It("[test_id:47674][reboot_required][slow][images][tier2] should be able to modify the configurable values under the NUMAResourcesOperator CR", func() {
+		It("[test_id:47674][reboot_required][slow][images][tier2] should be able to modify the configurable values under the NUMAResourcesOperator CR", Label("reboot_required", "slow", "images", "tier2"), func() {
```

Comment:
some labels are duplicated in the nested specs (mentioned in Describe(..) and in It(..)) for example "slow", it sounds better to have one only for instance on the parent spec if all the tests under it are slow, or on the test spec if not all children tests are classified as slow.

Reply:
This is true, but ginkgo is smart enough to de-duplicate labels, we can check with
```
./bin/e2e-nrop-serial.test -ginkgo.v -ginkgo.dry-run -ginkgo.no-color | less
```
I think it's fair to remove duplicate labels as we go, but if you feel it should be done in this PR I can do it as well

Reply:
ack, no need for now then